### PR TITLE
fix(webb-ui-components): Fixes Table component header misalignment

### LINKED ISSUE #2449: [BUG] Fix table component header alignment issue
### Overview

The table component header view is misaligned, causing the headers not to properly match the columns below them.

### Screenshots

CleanShot 2024-07-23 at 19 44 47@2x

CleanShot 2024-07-23 at 19 53 22@2x

### PR DESCRIPTION
**Pull request by devpavan04**
## Summary of changes

- Fixes Table component header misalignment.

### Proposed area of change

_Put an `x` in the boxes that apply._

- [ ] `apps/bridge-dapp`
- [ ] `apps/hubble-stats`
- [ ] `apps/stats-dapp`
- [x] `apps/tangle-dapp`
- [ ] `apps/testnet-leaderboard`
- [ ] `apps/faucet`
- [ ] `apps/zk-explorer`
- [x] `libs/webb-ui-components`

### Reference issue to close (if applicable)

- Closes #2449 

### Screen Recording

https://github.com/user-attachments/assets/d2114a3e-efcf-4c9d-b4d9-dfddf0947b0b

### CODE REVIEW

File: libs/webb-ui-components/src/components/Table/Table.tsx [hunk 1]
```diff
@@ -71,7 +70,7 @@ export const Table = <T extends RowData>({
                     }
                   >
                     {header.isPlaceholder ? null : (
-                      <>
+                      <div className="flex items-start justify-start">
```
Comment (AtelyPham) [inline libs/webb-ui-components/src/components/Table/Table.tsx:73]: The `className` in the `div` complicates header customization. Consider adding these utilities directly to the `<THeader>` component.

File: yarn.lock
Comment (AtelyPham) [inline yarn.lock:1]: I'm curious why the `yarn.lock` changed here. You didn't install any new dependencies.

**Review by AtelyPham — COMMENTED** [review]
Left some comments.

**Comment by devpavan04** [comment]
Addressed this in #2406

**Pull request closed without merging**